src/app/pages/send/send-crypto-asset-form/components/recipient-field.tsx

```diff
@@ -17,7 +17,7 @@ export function RecipientField({
       dataTestId={SendCryptoAssetSelectors.RecipientFieldInput}
       label="To"
       labelAction="Choose account"
-      name="recipientAddressOrBnsName"
+      name="recipient"
```

Comment:
Just leaving a comment that this change breaks bns names in the stacks recipient field. I just needed to do this for testing btc. I will address actually fixing it in my qa fixes branch. I believe it should just accept a name prop from each form or maybe the bns code can be written differently. I'll look.